plugin/pkg/admission/priority/admission.go

```diff
@@ -221,7 +222,7 @@ func (p *priorityPlugin) admitPod(a admission.Attributes) error {
 // validatePriorityClass ensures that the value field is not larger than the highest user definable priority. If the GlobalDefault is set, it ensures that there is no other PriorityClass whose GlobalDefault is set.
 func (p *priorityPlugin) validatePriorityClass(a admission.Attributes) error {
 	operation := a.GetOperation()
-	pc, ok := a.GetObject().(*scheduling.PriorityClass)
+	pc, ok := a.GetObject().(*schedulingv1beta1.PriorityClass)
```

Comment:
the admission plugin still gets an internal type.

Background: admission is part of the server chain.